src/v0/destinations/af/deleteUsers.js

```diff
@@ -32,7 +46,7 @@ const userDeletionHandler = async (userAttributes, config) => {
   if (!config?.apiToken || !(config?.appleAppId || config?.androidAppId)) {
     throw new ErrorBuilder()
       .setMessage(
-        "API Token and one of Apple ID or Android App Id are required fields for user deletion"
+        "[Appsflyer]::API Token and one of Apple ID or Android App Id are required fields for user deletion"
```

Comment:
Remove the prefix, please.

Reply:
mentioned reason in PR description: adding integration names to message to better debug at bugsnag

Reply:
Actually, it's not required. The destination name is already captured as part of the metadata and stack trace. Having this prefix pollutes the error messages shown to the user.

Reply:
![Screenshot 2022-12-20 at 6 08 56 PM](https://user-images.githubusercontent.com/25389167/208669424-d01bb34d-870e-4352-a9f5-c3f3d9e68e19.png)
Yeah but in bugsnag you have to open individual reports to get that. With this its easier to check the destination from error message. 
I know its not the convention we are aiming for but atleast will help in immediate monitoring later on as deemed we can remove these not a big lift